sherpa/optmethods/src/_saoopt.cc

```diff
@@ -606,8 +891,10 @@ static PyMethodDef WrapperFcts[] = {
   FCTSPEC(difevo, py_difevo),
   FCTSPEC(nm_difevo, py_difevo_nm),
   FCTSPEC(lm_difevo, py_difevo_lm),
+  FCTSPEC(cpp_lmder, py_lmder),
```

Comment:
Do we need `cpp_lmder` crreated/exported as I can't see a place where it is being used?